src/server/run_mir.cpp

```diff
@@ -162,6 +163,9 @@ extern "C" [[noreturn]] void fatal_signal_cleanup(int sig, siginfo_t* info, void
         constexpr size_t len = std::char_traits<char>::length(warning);
         [[maybe_unused]]auto n = write(STDERR_FILENO, warning, len);
     }
+    auto security_log = mir::security_fmt(mir::logging::Severity::error, "sys_crash", std::string{"Mir fatal signal received: "} + signum_to_string(sig)) + "\n";
```

Comment:
This attempts to allocate memory in a signal context. That is undefined behaviour (and we can avoid it here).

Also the reference above says "sys_crash...Level: WARN" is that prescriptive?